tests/mockserver_tests/mock_server_test_base.py

```diff
@@ -61,6 +61,27 @@ def aborted_status() -> _Status:
 def unavailable_status() -> _Status:
     error = status_pb2.Status(
         code=code_pb2.UNAVAILABLE,
+        message="Received unexpected EOS on DATA frame from server",
```

Comment:
This error message would only be returned in combination with an INTERNAL error code from Spanner.

Reply:
True, it was a mistype. Fixed and thank you.